module/VuFind/src/VuFindTest/Feature/FixtureTrait.php

```diff
@@ -64,7 +86,7 @@ protected function getFixtureDir($module = 'VuFind')
      */
     protected function getFixture($filename, $module = 'VuFind')
     {
-        $realFilename = realpath($this->getFixtureDir($module) . $filename);
+        $realFilename = $this->getFixturePath($filename, $module);
         if (!$realFilename || !file_exists($realFilename)
```

Comment:
This logic appears to be redundant since it has been refactored into getFixturePath above.

Reply:
I should have paid more attention. Fixed again. Third time lucky?